PLAN.md

```
@@ -1,160 +1,260 @@
+## Target Architecture
 
-Result: a `MonorepoContext | null`:
-```ts
-interface MonorepoContext {
-  root: string;             // absolute path
-  subpath: string;          // posix path, relative to root, e.g. 'apps/web'
-  packageManager: 'npm' | 'pnpm' | 'yarn' | 'bun';
-  // The detected framework still applies to the subpackage. The monorepo
-  // context just tells the rest of the pipeline how to install/build/run.
-}
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)